Words.playground/Contents.swift

```diff
@@ -46,10 +47,10 @@ extension WordCounter
             .filter { !$0.isEmpty }
     }
 
-    static func sanitizedWord(word: String) -> String?
+    static func sanitizedWord(word: String) -> String
     {
         let words = sanitizedStringComponents(word)
-        return words.count == 1 ? words.first : nil
+        return words.count == 1 ? words.first! : ""
```

Comment:
you can avoid the `!`.

`if let firstWord = words.first where words.count == 1 {
    return firstWord
} else {
    return ""
}